### DISCUSSION TITLE
Add option to filter more message types

### DISCUSSION THREAD

**Discussion by geeckon** [runelite/runelite#17848] [Ideas]
Currently the plugin "Chat filter" doesn't offer a way to filter out messages of type "FRIENDSCHATNOTIFICATION". These are game messages that appear under the "Channel" chat tab.
This would be useful to filter out some spammy messages in CoX.

A message of this type can easily be received for example by attempting to join a friends chat channel.

**Comment by Alexsuperfly** [comment]
FRIENDSCHATNOTIFICATION was added in 73c5762c0e45f8dfbe92d9aa11e862673a8119da as a filterable game chat message
